src/realm/array.cpp

```diff
@@ -1717,9 +1717,9 @@ void Array::alloc(size_t init_size, size_t width)
             // field in the header.
             size_t new_capacity_bytes = capacity_bytes * 2;
             if (new_capacity_bytes < capacity_bytes) // overflow detected, clamp to max
```

Comment:
I guess it is only useful on machines where `size_t` is 32 bits. We should keep it in for that case though. I think CI is all 64 bit, so it will be impossible to get code coverage on this line.